lib/controllers/users.js

```diff
@@ -0,0 +1,37 @@
+const { Router } = require('express');
+const { UserService } = require('../services/UserService');
+const authorize = require('../middleware/authorize');
+
+const ONE_DAY_IN_MS = 1000 * 60 * 60 * 24;
+
+module.exports = Router()
+  .post('/', [authorize], async (req, res, next) => {
```

Comment:
this passes your ```POST``` to ```/users``` to your ```authorize``` middleware, which means only users who are authorized are allowed to create users, but how can they be authorized if they haven't created a user?